packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -244,15 +158,17 @@ export class AuthenticatedEthereumAnchorService
     ethereumRpcUrl: string | undefined,
     logger: DiagnosticsLogger,
     pollInterval: number = DEFAULT_POLL_INTERVAL,
-    maxPollTime: number = MAX_POLL_TIME
+    maxPollTime: number = MAX_POLL_TIME,
+    enableLoop = false
```

Comment:
EthereumAnchorService defaults this to true, while AuthenticatedEthereumAnchorService and AnchorService defaults this to false. Why the difference?

Reply:
That's a mistake. Thank for noticing it.